lib/rules/web/ocm_console/ocm_workflow.xyaml

```diff
@@ -13,36 +13,36 @@ create_osd_cluster:
     action: overview_tab_loaded
 archive_cluster_from_list:
     action: cluster_list_page_loaded
-    action: try_to_archive_cluster
+    action: click_archive_button_on_cluster_list_page
     action: archive_dialog_loaded
     action: click_archive_button
     action: archive_succ_promt_message_displayed
     action: go_to_archived_cluster_page
     action: archived_page_loaded
 unarchive_cluster_from_list:
     action: archived_page_loaded
-    action: click_unarchive_cluster_button
+    action: click_unarchive_cluster_button_on_archived_cluster_list_page
     action: unarchive_dialog_loaded
     action: click_unarchive_button
     action: unarhive_succ_prompt_message_displayed
 archive_cluster_from_detail:
     action: cluster_list_page_loaded
-    action: switch_to_cluster_detail_page
-    action: ocp_cluster_detail_page_loaded
-    action: expand_detail_page_actions
-    action: click_detail_archive_cluster_button
+    action: go_to_cluster_detail_page
+    action: cluster_detail_page_loaded
+    action: expand_actions_on_cluster_detail_page
+    action: click_archive_button_on_cluster_detail_page
     action: archive_dialog_loaded
     action: click_archive_button
     action: archive_succ_promt_message_displayed
     action: archived_cluster_detail_page_loaded
 unarchive_cluster_from_detail:
     action: archived_page_loaded
-    action: switch_to_cluster_detail_page
+    action: go_to_cluster_detail_page
     action: click_detail_page_unarchive_cluster_button
     action: unarchive_dialog_loaded
     action: click_unarchive_button
     action: unarhive_succ_prompt_message_displayed
```

Comment:
typo error: `unarhive` 
Where is `unarhive_succ_prompt_message_displayed` and `archive_succ_promt_message_displayed`?